src/components/cards/menuCard/PEMenuCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { type ReactElement } from 'react';
+import PECarousel from '../../standard/carousel/PECarousel';
+import { Icon } from '../../standard/icon/Icon';
+import PeopleEatIcon from '../../standard/icon/PEIcon';
+
+export interface PEMenuCardProps {
+    onClick?: () => void;
+    title: string;
+    chefName: string;
+    chefPicture: string;
+    pricePerPerson: number;
+    description: string;
+    pictures: string[];
+    categories: string[];
+    kitchens: string[];
+}
+
+export default function PEMenuCard({
+    onClick,
+    title,
+    chefName,
+    chefPicture,
+    pricePerPerson,
+    description,
+    pictures,
+    categories,
+    kitchens,
+}: PEMenuCardProps): ReactElement {
+    return (
+        <div
+            onClick={onClick}
+            className="flex w-[580px] gap-3 flex-row p-3 box-border rounded-3 shadow-primary cursor-pointer active:shadow-orange hover:shadow-active"
+        >
+            <div className="flex rounded-3 overflow-hidden w-[220px] min-w-[220px] max-w-[220px] h-[220px] max-h-[220px] justify-center items-center bg-base">
+                {pictures && pictures?.length > 1 ? (
```

Comment:
With pictures no longer being optional, we also don't need optional chaining here:

```suggestion
                {pictures && pictures.length > 1 ? (
```

Reply:
Hm, do you think that pictures must be no optional?